reverse_proxy/templates/cran.conf.j2

```diff
@@ -0,0 +1,66 @@
+server {
+  listen 80;
+  server_name cran.{{zone}};
+
+  location /cran-fetch {
+    internal;
+    proxy_pass {{cran_mirror}};
+  }
+
+  location / {
+    # prevent cyclic location.capture
+    if ($request_uri ~ /cran-fetch) { break; }
+
+    # ensure we are only processing GET requests
```

Comment:
I'm not familiar with Lua (and I suspect not many others at HIC are 😄). Can you add some details comments, or a maybe a separate markdown doc? If it's written in the right way it could go straight into a paper.